doc/vaulted-shell.1.md

```diff
@@ -70,6 +70,15 @@ call. This allows the session to be cached for a longer duration than the assume
 roll call will allow (maximum of 1 hour). This also means that the assume role
 call is performed each time a new environment is spawned from the session.
 
+You can also specify the role without the proceeding context and Vaulted will attempt
```

Comment:
Just as above, "… without the proceeding context" seems like a strange way to state this… What if we stated this as: "You can also specify the just role name and Vaulted will attempt …"?